### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Artbsc1992**
# Achievements:

- Build a mobile version of a conference web page.
- Build a desktop version of a conference web page.
- Create elements dynamically using Javascript.
- Use CSS to make the web page look similar to the template.
- Use Javascript in order to create a menu on the mobile version.
### This is my video explaining the project. [Video](https://www.loom.com/share/25bd54cf463e4ffe87abb406759c3d9e?sharedAppSource=personal_library)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+The live demo [link](https://artbsc1992.github.io/conference-webpage/).
+
+## Author
+
+👤 **Arturo Hermida**
+
+- GitHub: [@Artbsc1992](https://github.com/Artbsc1992)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
Comment (gemmen29) [inline README.md:35]: - [x] Kindly add an acknowledgment to [Cindy Shin](https://www.behance.net/adagio07) the author of the original design as required by the Creative Commons License in your README file.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                      <a href="#">Join</a>
+                  </li>
+                  <li class="nav-links">
+                      <a href="#">Sponsor</a>
+                  </li>
+              </ul>
+          </div>
+        </div>
+      </nav>
+      </header>
```
Comment (gemmen29) [inline about.html]: Kindly add the black nav that exists in the Behance design in the `home` and `about` pages in the desktop version.
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Artbsc1992, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
Hi @Artbsc1992 , 

Great work on making the changes 👏 
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

well-1
Good luck! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**